### PR TITLE
feat: add rule trim

### PR DESCRIPTION
**Pull request by ayocodingit**
# Overview

- [feat: add rule trim](https://github.com/jabardigitalservice/dedi-core-service/commit/a6c6828b43f0c5c501d064124bb558d5d86f2b30)
- [feat: convert is_active boolean](https://github.com/jabardigitalservice/dedi-core-service/commit/3f28c7bd9fcf596b85e42f2c7680077284ee5928)

cc: @jabardigitalservice/jds-backend 

## Evidence
- title: feat: add rule trim
- project: Desa Digital
- participants: @ayocodingit

### CODE REVIEW

**Review by imamdev93 — APPROVED** [review]

**Review by indraprasetya154 — APPROVED** [review]

**Review by rachadiannovansyah — APPROVED** [review]

File: src/modules/testimonials/testimonial_test.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { createAccessToken } from '../../middleware/jwt'
 import app from '../../server'
 import { TestimonialEntity } from './testimonial_entity'
 
-const isActive = faker.random.arrayElement(['true', 'false'])
+const isActive = 'true'
```
Comment (sandisunandar99) [inline src/modules/testimonials/testimonial_test.ts:12]: btw mu tanya.. ini jadinya "true" bool atau string ya? kalo baca diatas isactive itu bool

Comment (ayocodingit) [reply src/modules/testimonials/testimonial_test.ts:12]: yang atas juga random element nya string bool mang, bukan type bool 😸

Comment (sandisunandar99) [reply src/modules/testimonials/testimonial_test.ts:12]: yoyo bungkus

**Review by sandisunandar99 — APPROVED** [review]

File: src/modules/testimonials/testimonial_test.ts (hunk 1 above)
Comment (ayocodingit) [reply src/modules/testimonials/testimonial_test.ts]: done yah mang [25a75db](https://github.com/jabardigitalservice/dedi-core-service/pull/177/commits/25a75dbe3bc79b56ae98c1510f5fa6ce5bc2cda9)

**Pull request merged**